packages/illustrated-message/src/illustrated-message.css

```diff
@@ -35,6 +35,36 @@ governing permissions and limitations under the License.
     text-transform: var(--spectrum-heading-quiet-2-text-transform, none);
     margin-top: 0;
     margin-bottom: 0;
+    color: var(
+        --spectrum-heading-page-title-text-color,
+        var(--spectrum-global-color-gray-700)
+    );
+}
+
+#description {
```

Comment:
Can we leave a comment that outlines why we need his override?

Reply:
Added!